### PR TITLE
Update install-with-docker.md

### PR DESCRIPTION
**Pull request by michaeldelago**
### Description

- Update some of the information around Docker's installation and remove extraneous info to match the repo.

## Thank you for contributing to the AzerothCore wiki.

Remember that the wiki is currently available in English and Spanish.

- https://www.azerothcore.org/wiki/home
- https://www.azerothcore.org/wiki/es/home

### CODE REVIEW

File: docs/install-with-docker.md [hunk 1]
```diff
@@ -315,22 +301,10 @@ This is an example of a fresh, empty AzerothCore server running with Docker on m
...
-
-Using Docker will have the same benefits as using virtual machines, but with much less overhead:
-
-![Docker containers vs Virtual machines](https://user-images.githubusercontent.com/75517/51078179-d4fec680-16b1-11e9-8ce6-87b5053f55dd.png)
-
-_AzerothCore running on macOS with Docker_
-![AzerothCore on macOS using Docker](https://user-images.githubusercontent.com/75517/51341229-2089e980-1a91-11e9-8d06-ebd5897552d4.png)
-
-_AzerothCore running on Windows 10 with Docker_
-![AzerothCore on Windows 10 with Docker](https://user-images.githubusercontent.com/75517/51561998-966ec600-1e80-11e9-939e-d522c71de459.png)
```
Comment (Helias) [inline docs/install-with-docker.md:328]: why did you remove this?

File: docs/install-with-docker.md [hunk 2]
```diff
@@ -3,35 +3,21 @@ tableofcontents: 1
...
 
-## Introduction
-
-Installing AzerothCore using Docker is a simplified procedure that has several benefits:
-
-- It's very easy! Docker will do all the dirty work for you.
-- It can be done in all operating systems where Docker is available (including **Windows**, **GNU/Linux**, **macOS**)
-- You don't need to install many dependencies (forget about _visual studio_, _cmake_, _mysql_, etc.. they are **NOT** required)
-- Forget about platform-specific bugs. When using Docker, AzerothCore will always run in **Linux-mode**. 
-- There are many other [benefits when using Docker](https://www.google.com/search?q=docker+benefits)
```
Comment (Helias) [inline docs/install-with-docker.md:18]: what was wrong with this?

File: docs/install-with-docker.md (hunk 1 above)
Comment (Kitzunu) [reply docs/install-with-docker.md:328]: Probably because it has nothing to do with building with docker

File: docs/install-with-docker.md [hunk 3]
```diff
@@ -3,35 +3,21 @@ tableofcontents: 1
 redirect_from: "/Install-with-Docker"
 ---
 
-# Install with Docker
+# Build with Docker
```
Comment (Kitzunu) [inline docs/install-with-docker.md:6]: This should match the file name.

Comment (michaeldelago) [reply docs/install-with-docker.md]: :upside_down_face:

File: docs/install-with-docker.md (hunk 2 above)
Comment (michaeldelago) [reply docs/install-with-docker.md:18]: I felt it was fluff and that this document should be describing how to deploy AzerothCore with Docker and that describing the benefits of it in this way was out of scope of the document. 

I'm not married to removing this section, I could just clean it up

1. Remove this bullet point. It doesn't actually tell the reader anything
2. Noting that it's cross platform/azerothcore in docker is supported (or at least, I've been supporting it for these platforms :D) cross platform is nice. Probably reword to clarify this point. 
3. remove parentheses and maybe clarify the original statement
4. if it's always running in "linux-mode", wouldn't platform specific-bugs for linux happen?
5. remove this bullet point. the reader can google "docker benefits" on their own

File: docs/install-with-docker.md (hunk 1 above)
Comment (michaeldelago) [reply docs/install-with-docker.md:328]: The images aren't scaled well so the page layout can be hard to read.

I think this document should describe building with docker and addressing the few differences in running the server with docker, but I don't think that we should describe the benefits or standard procedures for docker

**Comment by michaeldelago** [comment]
Added changes to reflect https://github.com/azerothcore/azerothcore-wotlk/pull/17419
